opm/input/eclipse/Schedule/Group/GuideRateModel.cpp

```diff
@@ -185,14 +189,18 @@ double GuideRateModel::eval(double oil_pot, double gas_pot, double wat_pot) cons
     }
 
 
-    double denom = this->B + this->C*std::pow(R1, this->D) + this->E*std::pow(R2, this->F);
-    /*
-      The values pot, R1 and R2 are runtime simulation results, so here
-      basically anything could happen. Quite dangerous to have hard error
-      handling here?
-    */
+    const double denom = this->B + this->C*std::pow(R1, this->D) + this->E*std::pow(R2, this->F);
     if (denom <= 0) {
-        throw std::range_error("Invalid denominator: " + std::to_string(denom));
+        OpmLog::warning("GUIDERATE_ZERO_DEONM",
+            fmt::format("GuideRateModel::eval: Denominator {}  <= 0, which is not allowed. "
```

Comment:
Since users will see this message, I think I'd prefer that we not use developer-centric names here.  Maybe something along the lines of
```
"GUIDERAT formula denominator {0:.6e} is non-positive..."
```